src/caf/space/ui.py

```diff
@@ -771,7 +770,7 @@ class NotebookApp(tk.Tk):
 
     def __init__(self):
         super().__init__()
-        # self.logger = logging.getLogger("SPACE")
+        self.logger = logging.getLogger("SPACE")
```

Comment:
Use `__package__` which will be 'caf.space'

```suggestion
        self.logger = logging.getLogger(__package__)
```